data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -351,7 +352,7 @@
         {VARIABLE_OP malin.experience add 4}
         [floating_text]
             x,y=$malin.x,$malin.y
-            text="<span color='#800080'>" + "+4 exp" + "</span>"
+            text=_ "<span color='#800080'>" + "+4 exp" + "</span>"
```

Comment:
I think the underscore should instead go after the first `+`.

Reply:
There was an wmllint magic comment to suppress this. I think for this case, it was to add at the end of the same line
` # wmllint: noconvert`

They are described in the comment text of the wmllint python script.

Reply:
done in ad68ef0